sdk/tdf_test.go

```diff
@@ -493,23 +465,36 @@ func TestTDF(t *testing.T) {
 
 		kasInfoList := test.kasInfoList
 		for index := range kasInfoList {
-			kasInfoList[index].URL = serverURL
+			kasInfoList[index].URL = s.serverURL
 			kasInfoList[index].PublicKey = ""
 		}
 
 		// test encrypt
-		testEncrypt(t, sdk, kasInfoList, plaintTextFileName, tdfFileName, test)
+		testEncrypt(s.T(), s.sdk, kasInfoList, plaintTextFileName, tdfFileName, test)
 
 		// test decrypt with reader
-		testDecryptWithReader(t, sdk, tdfFileName, decryptedTdfFileName, test)
+		testDecryptWithReader(s.T(), s.sdk, tdfFileName, decryptedTdfFileName, test)
 
 		// Remove the test files
 		_ = os.Remove(plaintTextFileName)
 		_ = os.Remove(tdfFileName)
 	}
 }
 
-func BenchmarkReader(b *testing.B) {
+/* func BenchmarkTDF(b *testing.B) {
```

Comment:
@sujankota Do we need this benchmark test? I was trying to update it to use testify suite but I don't think it will work. Also it didn't seem like we were doing the traditional golang benchmark test of 

```go
for i := 0; i < b.N; i++ {
  // Code to benchmark
}
```